stix_shifter/stix_transmission/stix_transmission.py

```diff
@@ -16,11 +16,22 @@ class StixTransmission:
     init_error = None
     
     def __init__(self, module, connection, configuration):
-        # pass
+        if module not in TRANSMISSION_MODULES :
+            raise NotImplementedError
+
+        dialect = None
```

Comment:
If the dialect is specific to Carbon Black, it may be better to push this logic down to the Connector if possible. I guess I'm not 100% sure what the module looks like before it gets split.